Source/Relativity.Test.Helpers/Relativity.Test.Helpers/TestHelper.cs

```diff
@@ -58,31 +62,35 @@ public static IHelper System()
 
 		public IDBContext GetDBContext(int caseID)
 		{
-			//You can create a new DBcontext using kCura.Data.RowDataGeteway until Relativity versions lower than 9.6.85.9
-			//kCura.Data.RowDataGateway.Context context = new kCura.Data.RowDataGateway.Context(SharedTestHelpers.ConfigurationHelper.SQL_SERVER_ADDRESS, string.Format("EDDS{0}", caseID == -1 ? "" : caseID.ToString()), SharedTestHelpers.ConfigurationHelper.SQL_USER_NAME, SharedTestHelpers.ConfigurationHelper.SQL_PASSWORD);
-			//return new DBContext(context);
-
-			//You can create a new DBcontext using DBContextHelper for Relativity versions equal to or greater than 9.6.85.9
-			DbContext context;
-
-			if (_alternateConfig != null)
-			{
-				context = new DbContext(this._alternateConfig.SqlServerAddress, $"EDDS{(caseID == -1 ? "" : caseID.ToString())}", this._alternateConfig.SqlUserName, this._alternateConfig.SqlPassword);
-			}
-			else
-			{
-				context = new DbContext(SharedTestHelpers.ConfigurationHelper.SQL_SERVER_ADDRESS, $"EDDS{(caseID == -1 ? "" : caseID.ToString())}", SharedTestHelpers.ConfigurationHelper.SQL_USER_NAME, SharedTestHelpers.ConfigurationHelper.SQL_PASSWORD);
-			}
-
-			return context;
+			throw new NotImplementedException();
 		}
 
 		public Guid GetGuid(int workspaceID, int artifactID)
 		{
-			var sql = "select ArtifactGuid from eddsdbo.ArtifactGuid where artifactId = @artifactId";
-			var context = GetDBContext(workspaceID);
-			var result = context.ExecuteSqlStatementAsScalar<Guid>(sql, new SqlParameter("artifactId", artifactID));
-			return result;
+			HttpClient httpClient = new HttpClient();

```

Comment:
I feel like line 70 to 75 should be wrapped up in a common method and re-used everywhere we want to make REST calls. I don't think this part of code will change for every REST call we make.